### PR TITLE
[tw] Localize Continuous Deployment (CD) for Traditional Chinese

### LINKED ISSUE #2457: [tw] Localize `Continuous Deployment (CD)` for Traditional Chinese
1. Localize `Continuous Deployment (CD)` for Traditional Chinese
2. https://glossary.cncf.io/continuous-deployment/

### PR DESCRIPTION
**Pull request by Fionn88**
### Describe your changes

Origin content from https://glossary.cncf.io/continuous-deployment/

### Related issue number or link (ex: `resolves #issue-number`)

resolves #2457 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/zh-tw/continuous-deployment.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+status: Completed
+category: 概念
+tags: ["應用程式", "方法論", ""]
+---
+
+## 是什麼 {#what-it-is}
+
+持續部署，通常縮寫為 CD，它直接將完成的軟體部署到正式環境，比[持續交付](/zh-tw/continuous-delivery/)更進一步。
+持續部署（CD）與[持續整合](/zh-tw/continuous-integration/)（CI）緊密相關，通常被稱為 CI/CD。
+CI 流程用於測試對特定應用程式的變更是否正確，而 CD 流程则自動部署組織測試環境的程式碼變更到正式環境。
```
Comment (johnlinp) [inline content/zh-tw/continuous-deployment.md:12]: 则 --> 則

File: content/zh-tw/continuous-deployment.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+## 解決的問題 {#problem-it-addresses}
+
+發布新軟體版本是一個勞動密集且容易出錯的過程。
+通常這也使組織希望盡量減少發布新版本的頻率，以避免正式環境發生事故並減少工程師需要在正常工作時間之外提供支援的次數。
+傳統的軟體部署模型讓組織陷入一個惡性循環，其中軟體發布的過程無法滿足組織在穩定性和功能迭代速度方面的需求。
+
+## 如何幫助我們 {#how-it-helps}
+
+通過自動化發布週期並強制組織更頻繁地將產品部署到正式環境中，CD 對維運團隊所做的就像 CI 對開發團隊所做的一樣。
```
Comment (johnlinp) [inline content/zh-tw/continuous-deployment.md:22]: "通過" --> "透過"

Comment (johnlinp) [inline content/zh-tw/continuous-deployment.md:22]: "CD 對維運團隊所做的就像 CI 對開發團隊所做的一樣" 這句，如果改成 "CD 為維運團隊所做的就像 CI 為開發團隊所做的一樣" 似乎更通順一點。或是也可以考慮翻譯成 "CD 之於維運團隊就像 CI 之於開發團隊一樣" 。

原文:

> By automating the release cycle and forcing organizations to release to production more frequently, CD does what CI did for development teams for operations teams.

Comment (Fionn88) [reply content/zh-tw/continuous-deployment.md:22]: 感覺這段 "CD 之於維運團隊就像 CI 之於開發團隊一樣" 比較通順，非常感謝

**Review by johnlinp — APPROVED** [review]
感謝!

**Review by hwchiu — APPROVED** [review]

**Pull request merged**